src/vivarium/framework/lookup/table.py

```diff
@@ -45,19 +47,24 @@ class LookupTable(ABC):
     def __init__(
         self,
         table_number: int,
+        return_type: type[T],
         key_columns: Sequence[str] = (),
         parameter_columns: Sequence[str] = (),
-        value_columns: Sequence[str] = (),
+        value_columns: list[str] | tuple[str, ...] | str = (),
         validate: bool = True,
     ):
         self.table_number = table_number
         """Unique identifier of the table."""
+        self.return_type: type[T] = return_type
```

Comment:
Can't this still be a scalar value? Or is it always converting that scalar value to a series LookupTable?